dev/MAINTAINING.md

```diff
@@ -26,71 +26,98 @@
 * If this is a major release check if any previously deprecated features are to
   be removed in this release.
 * Check pre-release APIs to be exposed, especially from streamly-examples
-* PVP is ahered to, do not change pre-release APIs in a minor release
+* PVP is adhered to, additionally do not change pre-release APIs in a
+  minor release
 
 * _Documentation_:
 
-    * README is updated
+    * README is updated, links in README are stable. Run `rg '/blob/'`
+      and `rg '/tree/'` to search for github links, replace `master` ref
+      with release tag where possible (e.g. streamly-examples). Ideally,
+      the distribution process should automatically replace these with
+      current stable references.
     * Haddock docs are consistent with the changes in the release
     * Tutorial has been updated for new changes
     * Documents in the `docs` directory are consistent with new changes
     * All combinators have time and space complexity annotations
     * All combinators have `since` annotation
+    * Modules that were renamed or APIs that moved to other modules should be
+      marked as "since" the current release.
+    * Check all released modules to ensure that they do not have any leftover
+      `Internal` or `Pre-release` annotations.
     * Unreleased combinators should be marked with `Pre-release` or
       `Internal` annotations
+    * Streamly homepage is updated with the release docs
+      * FileSystem.Event.Windows/Darwin modules require special handling
 
 * _Build and Test_:
 
     * All CIs are green
+    * Run tests using bin/test.sh to test with memory restrictions
     * Manually build and test for all flags (esp. `dev` flag) not covered by CIs
-    * Test latest GHC version with -O0 and -O1
+    * Test prime GHC version with -O0 and -O1
```

Comment:
What is the difference between `latest` and `prime`?
What is `prime`? Latest beta-ish?

Reply:
latest can have bugs that are not yet fixed e.g. the performance issues with ghc-9.0.1. Currently we call 8.10.5 as prime because 9.0 is not ready yet.